scalableminds/webknossos
# add meaningful error message for missing NML scale

### Steps to test:
- create explorative annotation
- delete its dataset on disk
- refresh datasets
- download annotation
- error message should say what the problem is

### Issues:
- contributes to #2622

------
